README.md

```diff
@@ -132,6 +132,66 @@ Handlers set up this way will have the `min`, `max`, and `current` write actions
 |             | `max`         | `int`      | The maximum bound for readings to be generated within. |
 |             | `current`     | `int`      | The static current reading value. |
 
+### Device Configurations
+
+In addition to allowing values to be set via writing to devices, the behaviors/data ranges
+for device readings may also be configured in the device configuration YAML for certain
+devices. These options should be set under a device's `data` configuration, e.g.
+
+```yaml
+version: 3
+devices:
+  - type: temperature
+    context:
+      model: emul8-temp
+    instances:
+      - info: Synse Temperature Sensor 1
+        data:
+          id: 1
+          min: 20
+          max: 40
+```
+
+The table below describes the supported configuration values for each type. None of the values
+are required to be set and falls back on sane defaults. Definitions for the settings follow:
```

Comment:
> None of the values are required to be set and falls back on sane defaults

Perhaps:
None of the values are required to be set, and if omitted the plugin falls back on sane defaults  OR
None of the values are required to be set as sane defaults are provided within the application